src/proto/tagpost_rpc.proto

```diff
@@ -44,3 +48,22 @@ message FetchCommentsUnderThreadRequest {
 message FetchCommentsUnderThreadResponse {
   repeated Comment comment = 1;
 }
+
+message AddThreadWithPrimaryTagRequest {
+  string tag = 1;
```

Comment:
I think we should also put a Thread object in this request, if we want to specify something like thread title, creator username later.

Reply:
Done.